source/material_model/composition_reaction.cc

```diff
@@ -243,6 +258,20 @@ namespace aspect
           (compositional_delta_rho_2 != 0))
         this->model_dependence.density |= NonlinearDependence::compositional_fields;
     }
+
+
+    template <int dim>
+    void
+    CompositionReaction<dim>::create_additional_named_outputs (MaterialModel::MaterialModelOutputs<dim> &out) const
+    {
+      if (out.template get_additional_output<ReactionRateOutputs<dim> >() == NULL)
```

Comment:
does this mean (here and in all other places) that you always create these outputs even when we don't use operator splitting? That doesn't seem right.